docs: Separate section on ingesting MVDs in migration guide (#17109)
apache/druid: docs/release-info/migr-mvd-array.md

* For SQL-based batch ingestion, include the [query context parameter](../multi-stage-query/reference.md#context-parameters) `"arrayIngestMode": "array"` and reference the relevant array type (`VARCHAR ARRAY`, `BIGINT ARRAY`, or `DOUBLE ARRAY`) in the [EXTEND clause](../multi-stage-query/reference.md#extern-function) that lists the column names and data types.
For examples, see [Ingesting arrays: SQL-based ingestion](../querying/arrays.md#sql-based-ingestion).

## How to ingest MVDs

You can't mix arrays and MVDs in the same column.

> If you need to continue to use MVDs, modify your ingestion to include the [ARRAY_TO_MV](../querying/sql-functions.md#array_to_mv) function.

Reviewer: ```suggestion
If you need to continue to use MVDs, use the [ARRAY_TO_MV](../querying/sql-functions.md#array_to_mv) function when you ingest data.
```

This ensures that VARCHAR ARRAYS are stored as MVDs rather than arrays of strings.

> Ingest MVDs explicitly so that you can continue to use existing queries that involve MVDs without incompatible behavior from arrays.

Reviewer: ```suggestion
To continue using MVDs in your existing queries, you need to ingest MVDs explicitly since arrays and MVDs behave differently.
```
